app/controllers/notes_controller.rb

```diff
@@ -55,10 +55,56 @@ def destroy
       track_destroyed(@note)
       redirect_to node_path(@node), notice: 'Note deleted'
     else
-      redirect_to node_note_path(@node, @note), alert: 'Could not delete node'
+      redirect_to node_note_path(@node, @note), alert: 'Could not delete note'
     end
   end
 
+  def multi_destroy
+    @notes = Note.where(
+      'node_id = :node AND id in (:ids)',
+      node: @node,
+      ids: params[:ids]
+    )
+
+    if @notes.any?
+      @job_logger = Log.new(uid: (Log.maximum(:uid) || 0) + 1)
```

Comment:
I've seen this pattern in multiple places around the code, can we encapsulate it in the initializer and make it the default behaviour if no `:uid` is passed?